server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v64/DbVersion64.java

```diff
@@ -28,6 +28,11 @@
   public void addSteps(MigrationStepRegistry registry) {
     registry
       .add(1600, "Add Projects.TAGS", AddTagsToProjects.class)
-      .add(1601, "Set PROJECTS.COPY_COMPONENT_UUID on local views", SetCopyComponentUuidOnLocalViews.class);
+      .add(1601, "Set PROJECTS.COPY_COMPONENT_UUID on local views", SetCopyComponentUuidOnLocalViews.class)
+      .add(1602, "Add RULES_PROFILES.ORGANIZATION_UUID", AddQualityProfileOrganizationUuid.class)
+      .add(1603, "Set RULES_PROFILES.ORGANIZATION_UUID to default", SetQualityProfileOrganizationUuidToDefault.class)
+      .add(1604, "Set RULES_PROFILES.ORGANIZATION_UUID to not nullable", SetQualityProfileOrganizationUuidToNotNullable.class)
+      .add(1605, "Let RULES_PROFILES.KEE not be unique anymore", LetQualityProfileKeyNotBeUniqueAnymore.class)
```

Comment:
Strange name ("let"). It should be more "factual" : `DropUniqueIndexOnQualityProfileKey`